featuretools/computational_backends/calculate_feature_matrix.py

```diff
@@ -751,3 +751,18 @@ def update_progress_callback_parameters(progress_bar, previous_progress):
     progress_percent = (progress_bar.n / progress_bar.total) * 100
     time_elapsed = progress_bar.format_dict["elapsed"]
     return (update, progress_percent, time_elapsed)
+
+
+def init_ww_and_concat_fm(feature_matrix, ww_init_kwargs):
+    for fm in feature_matrix:
+        fm.ww.init(**ww_init_kwargs)
```

Comment:
We also initialize the feature matrix after concat. Is this init still necessary?

Reply:
Those inits are to ensure each smaller matrix has compatible types for when we do the larger concat

as an example, there could be what's supposed to be a `BooleanNullable` column and the smaller matrices have these values:

series 1: [True, False], bool dtype
series 2: [1.0, np.nan], float dtype
seires 3: [np.nan, np.nan], float dtype

and after the concat we get

series: [True, False, 1.0, np.nan, np.nan, np.nan], object dtype

and when try to cast to  `BooleanNullable` pandas is unable to convert the type